src/cli/repo/s3.go

```diff
@@ -73,7 +74,7 @@ corso repo init s3 --bucket my-bucket
 corso repo init s3 --bucket my-bucket --prefix my-prefix
 
 # Create a new Corso repo in an S3 compliant storage provider
-corso repo init s3 --bucket my-bucket --endpoint https://my-s3-server-endpoint`
+corso repo init s3 --bucket my-bucket --endpoint my-s3-server-endpoint`
```

Comment:
Haha, good catch!

Reply:
Yupp! also will it apply to `repo connect` too?

Reply:
Yup, it should be dropped from there too.